### PR TITLE
Move preview to tab pane instead of workflow page

### PR DESCRIPTION
**Pull request by alexfmpe**

### CODE REVIEW

**Review by benkolera — APPROVED** [review]

File: frontend/src/Frontend/UI/DeploymentSettings.hs [hunk 1]
```diff
@@ -293,26 +308,29 @@ buildDeployTabs mUserTabName controls = mdo
   where
     userTabs = maybeToList mUserTabName
     stdTabs = [DeploymentSettingsView_Cfg, DeploymentSettingsView_Keys]
-    availableTabs = userTabs <> stdTabs
+    withPreview = if includePreviewTab then [DeploymentSettingsView_Preview] else []
+    availableTabs = userTabs <> stdTabs <> withPreview
 
 defaulTabViewProgressButtonLabel :: DeploymentSettingsView -> Text
-defaulTabViewProgressButtonLabel DeploymentSettingsView_Keys = "Preview"
```
Comment (tomsmalley) [inline frontend/src/Frontend/UI/DeploymentSettings.hs:299]: Should this not still say "Preview"?

File: frontend/src/Frontend/UI/Dialogs/Signing.hs [hunk 2]
```diff
@@ -70,6 +70,7 @@ uiSigning appCfg ideL signingRequest onCloseExternal = do
           , _deploymentSettingsConfig_gasLimit = _signingRequest_gasLimit signingRequest
           , _deploymentSettingsConfig_caps = Just $ _signingRequest_caps signingRequest
           , _deploymentSettingsConfig_extraSigners = fromPactPublicKey <$> fromMaybe [] (_signingRequest_extraSigners signingRequest)
+          , _deploymentSettingsConfig_includePreviewTab = False
```
Comment (tomsmalley) [inline frontend/src/Frontend/UI/Dialogs/Signing.hs]: Why?

**Review by tomsmalley — CHANGES_REQUESTED** [review]

File: frontend/src/Frontend/UI/DeploymentSettings.hs (hunk 1 above)
Comment (alexfmpe) [reply frontend/src/Frontend/UI/DeploymentSettings.hs:299]: Maybe? The tab order makes it clear “Preview” follows. It seems only the last tab is given a special label to signify moving on

File: frontend/src/Frontend/UI/Dialogs/Signing.hs (hunk 2 above)
Comment (alexfmpe) [reply frontend/src/Frontend/UI/Dialogs/Signing.hs]: Why a default or why this default?

EDIT: Whoa what was I even looking at? Disregard this comment

Comment (tomsmalley) [reply frontend/src/Frontend/UI/Dialogs/Signing.hs]: Signing requests should have preview screens - unless I've missed communication from Kadena

Comment (alexfmpe) [reply frontend/src/Frontend/UI/Dialogs/Signing.hs]: Not sure - @mankyKitty ?

Comment (mankyKitty) [reply frontend/src/Frontend/UI/Dialogs/Signing.hs]: Signing requests should show the preview screen.

File: frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs [hunk 3]
```diff
@@ -150,6 +153,7 @@ uiAddVanityAccountSettings ideL mChainId initialNotes = Workflow $ do
             , _deploymentSettingsConfig_gasLimit = Nothing
             , _deploymentSettingsConfig_caps = Nothing
             , _deploymentSettingsConfig_extraSigners = []
+            , _deploymentSettingsConfig_includePreviewTab = False
```
Comment (tomsmalley) [inline frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs]: Sholud this also use `includePreviewTab` from above?

Comment (alexfmpe) [reply frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs]: It does now

File: frontend/src/Frontend/UI/Dialogs/Signing.hs (hunk 2 above)
Comment (alexfmpe) [reply frontend/src/Frontend/UI/Dialogs/Signing.hs]: It does now

**Pull request merged**